Test/refman/top-level-export.dfy

```diff
@@ -0,0 +1,11 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+module M {}
+const x := 10
+
+export provides x  // No exports allowed at top-level
```

Comment:
A bit confused on this. If this is not allowed, why is the expected file saying "Dafny program verifier finished with 0 verified, 0 errors"